.github/workflows/docker.yaml

```
@@ -0,0 +1,62 @@
+name: Publish Docker
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image (optional)'
+        required: false
+  create:
```

I think it's `push`? 
```suggestion
  push:
```

---

`create` worked fine in the original project.